tests/common/reboot.py

```diff
@@ -76,7 +77,15 @@
         "wait": 120,
         "cause": "Watchdog",
         "test_reboot_cause_only": True
-    }
+    },
+    REBOOT_TYPE_SAI_WARM: {
+        "command": "sai_warmboot.sh",
```

Comment:
Do we have `sai_warmboot.sh` on DUT? If no, exception is raised when we call it.